VERSION

```diff
@@ -84,32 +84,32 @@ date="Unreleased developer copy"
 # Version numbers are described in the Libtool current:revision:age
 # format.
 
-libmpi_so_version=0:0:0
-libmpi_cxx_so_version=0:0:0
-libmpi_mpifh_so_version=0:0:0
-libmpi_usempi_tkr_so_version=0:0:0
-libmpi_usempi_ignore_tkr_so_version=0:0:0
-libmpi_usempif08_so_version=0:0:0
-libopen_rte_so_version=0:0:0
-libopen_pal_so_version=0:0:0
-libmpi_java_so_version=0:0:0
-liboshmem_so_version=0:0:0
-libompitrace_so_version=0:0:0
+libmpi_so_version=50:0:10
+libmpi_cxx_so_version=50:0:10
+libmpi_mpifh_so_version=50:0:10
+libmpi_usempi_tkr_so_version=50:0:10
+libmpi_usempi_ignore_tkr_so_version=50:0:10
+libmpi_usempif08_so_version=50:0:10
+libopen_rte_so_version=50:0:10
+libopen_pal_so_version=50:0:10
+libmpi_java_so_version=50:0:10
+liboshmem_so_version=50:0:10
+libompitrace_so_version=50:0:10
 
 # "Common" components install standalone libraries that are run-time
 # linked by one or more components.  So they need to be versioned as
 # well.  Yuck; this somewhat breaks the
 # components-don't-affect-the-build-system abstraction.
 
 # OMPI layer
-libmca_ompi_common_ompio_so_version=0:0:0
+libmca_ompi_common_ompio_so_version=50:0:9
```

Comment:
@bwbarrett Why on earth did we make OMPI's C value be 41 in v3.0.0?  😱 (this is just a rhetorical question)

Also, I notice that `ompi/mca/common/monitoring` has no version number here.  Indeed, it doesn't look like it is versioned at all, which seems problematic.

This PR should still go through, but I think we need another PR that adds a version number for the common monitoring library.

Reply:
We didn't; see the commit comment.  There was a backwards-incompatible change in common_ompio in 3.0.1.  3.1 includes that change, so it should only be backwards compatible to 3.0.1, not 3.0.0 (well, the shared libraries built by those versions, but you know what I mean).

Good catch on the monitoring change; thanks for making that.